src/astro/databases/google/bigquery.py

```diff
@@ -58,3 +58,7 @@ def load_pandas_dataframe_to_table(
             chunksize=chunk_size,
             project_id=self.hook.project_id,
         )
+
+    def replace_illegal_columns_chars(self, char) -> str:
```

Comment:
What do you think if, within the Database, we only defined:
* `illegal_column_name_chars: List[chr]`
* `illegal_column_name_char_replacement: chr`

And we allowed the Operator to use those and replace them, as necessary? I understand that the replacement happens in the context of loading a JSON which has to be normalized - so perhaps the reserve itself should be kept to the load operation, whereas this bit - which is Database-specific, is held in the DB.